demos/shadow/shadow_demo_main/shadow_demo_main.c

```diff
@@ -768,7 +768,7 @@ int main( int argc,
              *
              * #define SHADOW_TOPIC_MAX_LENGTH  (256U)
              *
-             * ShadowStatus_t shadowStatus = SHADOW_STATUS_SUCCESS;
+             * ShadowStatus_t shadowStatus = SHADOW_SUCCESS;
```

Comment:
can you please update SHADOW_TOPIC_STRING_TYPE_UPDATE_DELTA to ShadowTopicStringTypeUpdateDelta on line 778 as well

Reply:
Good catch, thanks!